tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -3861,7 +3861,7 @@ All of JSX and Flow syntax is supported. In fact, the test suite in \`tests/flow
 _is_ the entire Flow test suite and they all pass.
 
 Prettier also supports [TypeScript](https://www.typescriptlang.org/), CSS,
-[LESS](http://lesscss.org/), [SCSS](http://sass-lang.com),
```

Comment:
We don't need to update test cases/snapshots. 😂

Reply:
But there is no harm in it.. right?

Reply:
Yeah, just 😂.